upload-symbols.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+if which sentry-cli >/dev/null; then
+    if [ -f .env ] && grep -q "^SENTRY_ORG=" .env && grep -q "^SENTRY_PROJECT=" .env; then
+        export $(grep -v '^#' .env | sed '/^\s*$/d' | xargs)
+    else
+        echo "[ERROR] .env does not exist or does not have SENTRY_ORG and SENTRY_PROJECT defined"
+        exit 1
+    fi
+    if [ ! -n "$SENTRY_AUTH_TOKEN" ]; then
+        if [ -f ~/.sentryclirc ]; then
+            export SENTRY_AUTH_TOKEN=$(grep -oE "token=([^\n\r]*)$" ~/.sentryclirc | cut -d'=' -f2)
+            echo "Using SENTRY_AUTH_TOKEN from .sentryclirc."
+        fi
+        if [ -f ~/.zshrc ] && grep -q "export SENTRY_AUTH_TOKEN" ~/.zshrc; then
+            grep -m 1 "export SENTRY_AUTH_TOKEN" ~/.zshrc > /tmp/ios.sentry-build.tmp && source /tmp/ios.sentry-build.tmp && rm /tmp/ios.sentry-build.tmp
+            echo "Using SENTRY_AUTH_TOKEN from .zshrc."
+        fi
```

Comment:
Removed the `else` clauses to log where it's not found. I found it more obvious to follow along when it tells me which one it's actually using.